### ISSUE TITLE
Antialiasing and Antistropic Filtering options don't work

### ISSUE THREAD

**Issue by feilen** [mabrowning/minecrift#143]
If you click "Antialiasing" or "Antistropic filtering" they don't change values, AA remains as "OFF" and AF doesn't even display a number.

Could be an issue with Optifine.

**Comment by StellaArtois** [comment]
Yup, that'll be Optifine. That said, I'll port to the latest Optifine soon
- there's been a large number of fixes added to it lately.

**Comment by feilen** [comment]
Alright cool. Considering my machine's power, I'd prefer upscaling at 1.0x, no SSAA, and just typical AA for smoothness. My machine's not underpowered but even on 'normal' rendering distance it's quite stuttery.
